src/loaders/types.d.ts

```diff
@@ -9,6 +9,7 @@ export type MobileArticleVideoLoaderProps = MobileArticleVideoPlayerProps | unde
 export type CanonicalVideoLoaderProps = CanonicalVideoPlayerProps | undefined;
 
 interface MWConfig {
+	// eslint-disable-next-line  @typescript-eslint/no-explicit-any
 	get: (key: string) => any;
```

Comment:
Why not `unknown` in place of `any`?

Reply:
```suggestion
	get: <T = unknown>(key: string) => T;
```

Reply:
Well, because this is object can truly be anything. With @nandy-andy we agreed to use a suppress since this any type is there on purpose.

Reply:
It's not a strong opinion, though :-) While live-code-reviewing we briefly discussed if we shouldn't put `unknown` here and we came to the conclusion @David-Iacovone mentioned ;-)

I think the image above this blog post explains very well why @pbartkowiak-fandom and @jbienkowski-fandom encourage us to use `unknown` here: https://dmitripavlutin.com/typescript-unknown-vs-any/ Let's do it. I mean, with the suggestion from Jakub it's just a matter of clicking "Commit suggestion" :D 

Thanks for feedback and short discussion! :-)